index.html

```diff
@@ -31,6 +31,7 @@ <h1>Calculators</h1>
   <a href="./calculators/tip-calculator">Tip Calculator</a><br />
   <a href="./calculators/PX-to-EM-converter/index.html">Pixel to Em Converter </a><br />
   <a href="./calculators/count-by-weight/index.html">Count by Weight</a><br />
+  <a href="./calculators/discount-calculator/index.html">Discount Calculator</a>
```

Comment:
@anuamloh07 need a `<br />` end of line

Reply:
I added the accepted tag in the PR, but I need you to correct this to do the merge. Thanks!

Reply:
I fix this